packages/common/src/search/_internal/portalSearchGroupMembers.ts

```diff
@@ -37,15 +37,29 @@ export async function portalSearchGroupMembers(
   query: IQuery,
   options: IHubSearchOptions
 ): Promise<IHubSearchResponse<IHubSearchResult>> {
-  if (!query.properties?.groupId) {
+  // Requires that the query have a filter with a group predicate
+  let groupId: string;
+  query.filters.forEach((filter) => {
+    filter.predicates.forEach((predicate) => {
+      const prop = getProp(predicate, "group");
+      if (Array.isArray(prop)) {
+        // get first entry from array
+        groupId = prop[0];
+      }
+      if (typeof prop === "string") {
+        // get the value as a string
+        groupId = prop;
+      }
+    });
```

Comment:
Minor thing, but since predicate entries can also be `IMatchOptions` object, it may be useful to add an extra `else` that merely logs a warning that `IMatchOptions` are not valid for `group` predicates on `groupMember` searches